src/signer.ts

```diff
@@ -80,26 +85,34 @@ export class Signer {
   #accountsPrompt: AccountsPrompt | undefined;
   #consentMessagePrompt: ConsentMessagePrompt | undefined;
   #callCanisterPrompt: CallCanisterPrompt | undefined;
+  #owner: IdentityNotAnonymous | null;
 
   // TODO: improve implementation to avoid an unexpected misusage in the future where an issue in the code would lead the busy flag to be reset to idle while effectively still being busy
   #busy = false;
 
   readonly #signerService = new SignerService();
 
-  private constructor(options: SignerOptions) {
+  private constructor(options: SignerInitOptions) {
     this.#signerOptions = options;
+    this.#owner = null;
 
     window.addEventListener('message', this.onMessageListener);
   }
 
+  setOwner = (owner: IdentityNotAnonymous) => {
```

Comment:
- Can you add the missing JS docs.

- Should we use an object as parameter? {owner}: {owner: IdentityNotAnonymous}?